docusaurus/docs/manage/02-router/01-overview.md

```diff
@@ -0,0 +1,63 @@
+---
+id: router-deploying-overview
+title: Deployment Overview
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import styles from './styles.module.css';
+
+In this article, we are describing the process of router deployment and what attributes can be updated, removed, added after the deployment is completed.  
+
+## Components of Deployment
+
+:::info Notes
+Currently, the DNS name or IP address can not be changed after the deployment, because it requires to update the `Sans` section of the router server certificate. To do that one would need to re-enroll the router, which can not be done without creating a new router.
+:::
+
+<Tabs>
+<TabItem value="Identity" label="Identity" attributes={{className: styles.green}}>

```

Comment:
I'd really rather see these not presented as tabs so that a reader can just scroll up and down and so that the right nav bar is useful to locate the sections if needed